src/core/defaults.ts

```diff
@@ -62,7 +62,9 @@ const DEFAULTS: Partial<Configuration> = {
       productCount: 0
     }
   },
-  network: {},
+  network: {
+    https: typeof window !== 'undefined' && window.location.protocol === 'https:'
```

Comment:
we should use the `WINDOW()` in utils, we don't have to handle the case where `window` is undefined for now as we assume to always be running in the browser

Reply:
The tests fail if I don't check that window is defined, since `WINDOW()` happens at import time so it's not capable of being stubbed.

Reply:
ah heck, okay, then this is approved